scripts/simapp/start.sh

```diff
@@ -4,6 +4,8 @@ command -v shellcheck > /dev/null && shellcheck "$0"
 
 TENDERMINT_PORT_GUEST="26657"
 TENDERMINT_PORT_HOST="26657"
+API_PORT_GUEST="1317"
+API_PORT_HOST="1318"
```

Comment:
Why not 1317 on the host? It would be what everyone expects

Reply:
wasmd runs on 1317. Since the simd API is only used for debugging, I thought it is better to move this one port up.

Reply:
Fair enough, but then we should probably bump up the tendermint rpc port as well?

Reply:
Possible, but not needed as long as we don't connect to wasmd Tendermint, which we don't:

```sh
LCD_API_PORT_GUEST="1317"
LCD_API_PORT_HOST="1317"
# Tendermint port (26657) and p2p port (26656) are not exposed since we don't need them for testing
```

I'll add a port overview for this repo …